/**:
ros__parameters:
  link_1: 100
  link_2: 80
  link_3: 50
  joint_degrees: 100